spec/openapi.yaml

```diff
@@ -85,150 +365,257 @@ paths:
+          $ref: '#/components/schemas/Category_data'
+        has_next:
+          type: integer
+        has_previous:
+          type: integer
+        number_of_pages:
+          type: integer
+        page:
+          type: integer
+        records_per_page:
+          type: integer
         status:
           type: string
-    inline_response_200_data:
+        status_code:
+          type: integer
+        total_count:
+          type: integer
+    Category_data:
+      type: array
+      items:
+        type: object
+        properties:
+          id:
+            type: integer
+          name:
+            type: string
+
+    Language:
       type: object
       properties:
-        name:
+        apiVersion:
           type: string
-        id:
+        data:
+          $ref: '#/components/schemas/Language_data'
+        has_next:
+          type: integer
+        has_previous:
+          type: integer
+        number_of_pages:
+          type: integer
+        page:
           type: integer
-    inline_response_200:
-      type: object
-      properties:
         records_per_page:
           type: integer
-        apiVersion:
+        status:
           type: string
         status_code:
           type: integer
+        total_count:
+          type: integer
+    Language_data:
+      type: object
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+    
+    SearchResponse:
+      type: object
+      properties:
+        apiVersion:
+          type: string
         data:
           type: array
           items:
-            $ref: '#/components/schemas/inline_response_200_data'
+            type: object
+            $ref: '#/components/schemas/Resource_data'
         number_of_pages:
           type: integer
+        page:
+          type: integer
+        records_per_page:
+          type: integer
+        status:
+          type: string
+        status_code:
+          type: integer
         total_count:
           type: integer
+    FilteredResourcesResponse:
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        data:
+          type: array
+          items:
+            type: object
+            $ref: '#/components/schemas/Resource_data'
         has_next:
-          type: boolean
-        has_prev:
-          type: boolean
+          type: integer
+        has_previous:
+          type: integer
+        number_of_pages:
+          type: integer
         page:
           type: integer
+        records_per_page:
+          type: integer
         status:
           type: string
+        status_code:
+          type: integer
+        total_count:
+          type: integer
+    ApiKeyResponse:
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        data:
+          $ref: '#/components/schemas/ApiKeyResponse_data'
+        status:
+          type: string
+        status_code:
+          type: integer
```

Comment:
This shows as `0` in the example code. The status_code is the http status code, so it should be `200` in the success example and `401` in the unauthorized example. I'm not sure if we can specify what defaults should be, but I hope we can because it's somewhat inaccurate and confusing

Reply:
I think it just looks at the schema definition and since status code is supposed to be an integer, it just shows 0? I will update the definition and add actual examples for the requests/responses. I have them on some of the endpoints and not others.